src/docs/GetStarted/Designers/aboutTheSystem/index.mdx

```diff
@@ -3,12 +3,12 @@ title: 'About the system'
 order: 1
 ---
 
-_As designers_, the **Mozaic design system** provide you with a lot of ready to use **patterns**.
+_As designers_, the **Mozaic design system** provides you with a lot of ready to use **patterns**.
```

Comment:
_As designers_, **Mozaic Design System** provides you with a lot of ready to use **patterns**.